From: Brian May <brian@linuxpenguins.xyz>
Date: Wed, 17 May 2017 17:03:25 +1000
Subject: Drop test_too_many_fields tests

Producing the TooManyFields exception was a sign of a limitation in the
number of datetimefields we could parse. It doesn't appear to be an
error that we are not generating this exception, rather it seems the
limitations that once existed have been removed.

For more information see:
https://github.com/jenisys/parse_type/issues/7
---
 tests/test_parse_type_parse.py | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/tests/test_parse_type_parse.py b/tests/test_parse_type_parse.py
index 23725f2..9374cd5 100644
--- a/tests/test_parse_type_parse.py
+++ b/tests/test_parse_type_parse.py
@@ -605,10 +605,6 @@ class TestParse(unittest.TestCase):
         self.assertNotEqual(r, None)
         self.assertEqual(r.fixed[21], 'spam')
 
-    def test_too_many_fields(self):
-        p = parse.compile('{:ti}' * 15)
-        self.assertRaises(parse.TooManyFields, p.parse, '')
-
 
 class TestSearch(unittest.TestCase):
     def test_basic(self):
